packages/contracts-bedrock/src/dispute/interfaces/IDisputeGame.sol

```diff
@@ -1,11 +1,11 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.15;
 
-import "../../libraries/DisputeTypes.sol";
-
 import { IBondManager } from "./IBondManager.sol";
 import { IInitializable } from "./IInitializable.sol";
 
+import "../../libraries/DisputeTypes.sol";
```

Comment:
Can we also not use the top-level import style here?

Reply:
Missed this one, thanks king
![image](https://github.com/ethereum-optimism/optimism/assets/8406232/26626453-bc03-4ed2-b489-1487e2c6ddb6)